src/app-airtable.js

```diff
@@ -42,26 +42,6 @@ class Client {
   }
 }
 
-// Function to transform Airtable records into Client entities
-function transformToClientEntities(records) {
-  return records.map((record) => {
-    const fields = record.fields;
-    return new Client(
-      record.id,
-      fields.account_user_id ? fields.account_user_id[0] : null,
-      fields.client_coach_user_id ? fields.client_coach_user_id[0] : null,
-      fields.user_ids_shared_with ? fields.user_ids_shared_with[0] : [],
```

Comment:
This was accessing a non-existent column.  This means that the view was rendering budgets without caring if they were shared with the coach. @zachwhelchel